ckcore/core/__main__.py

```diff
@@ -109,7 +109,7 @@ async def on_start() -> None:
                 "created_at": to_json(system_data.created_at),
                 "system": platform.system(),
                 "platform": platform.platform(),
-                "inside_docker": in_docker,
+                "inside_docker": inside_docker,
```

Comment:
What is this being used for and how does it relate to binding on any interface?

Reply:
somehow unrelated - the variable existed. I just renamed it. Does not change any logic.